app/models/user.rb

```diff
@@ -1,4 +1,8 @@
-class User < ActiveRecord::Base
+qclass User < ActiveRecord::Base
+  has_many :activities
+  validates :uid, presence: true
+  validates :provider, presence: true, inclusion: {in: %w(twitter), allow_blank: true}
+  validates :user_id, presence: true 
```

Comment:
ah i though that was what we were going to change on the db yesterday. 

so we can undo that when we talk later then....... 

got it - it was late last night..........i was too tired to be working 

----- Original Message -----

From: "Levi Cole" <notifications@github.com> 
To: "TeriPastorino/KnoWater" <KnoWater@noreply.github.com> 
Cc: "TeriPastorino" <tpasto@comcast.net> 
Sent: Wednesday, May 20, 2015 8:40:46 AM 
Subject: Re: [KnoWater] Shower log (#8) 



In app/models/user.rb : 
> @@ -1,4 +1,8 @@
> -class User < ActiveRecord::Base
> +qclass User < ActiveRecord::Base
> +  has_many :activities
> +  validates :uid, presence: true
> +  validates :provider, presence: true, inclusion: {in: %w(twitter), allow_blank: true}
> +  validates :user_id, presence: true 


there is no user_id on this table, if there is, there shouldn't be. 

— 
Reply to this email directly or view it on GitHub .